topics/admin/tutorials/heterogeneous-compute/tutorial.md

```diff
@@ -254,34 +255,9 @@ We also need to create the dependency resolver file so pulsar knows how to find
 >
 > 2. Log in to the machines and have a look in the `/mnt/pulsar` directory. You will see the venv and config directories. All the config files created can be perused.
```

Comment:
> All the config files created can be perused.

We don't want to modify Pulsar config files outside of Ansible, I think.

Reply:
```suggestion
> 2. Log in to the machines and have a look in the `/mnt/pulsar` directory. You will see the venv and config directories. All the config files created by Ansible can be perused.
```

Reply:
@natefoo can you click the button on these requests? I don't seem to have permissions for that, for some reason, despite spending past weeks pushing to the branch.